sdk/keyvault/azure-keyvault-certificates/README.md

```diff
@@ -38,14 +38,13 @@ See [azure-identity][azure_identity] documentation for more information about ot
 #### Create a client
 After configuring your environment for the [DefaultAzureCredential][default_cred_ref] to use a suitable method of authentication, you can do the following to create a certificate client (replacing the value of `vault_url` with your vault's URL):
```

Comment:
We would need to adjust comments that refer to the old code snippets, like this one. Like Kashif mentioned, I wonder what the best way to include imports in these snippets -- without having repeated imports in our samples -- would be...
```suggestion
After configuring your environment for the [DefaultAzureCredential][default_cred_ref] to use a suitable method of authentication, you can do the following to create a certificate client (replacing the value of `VAULT_URL` with your vault's URL):
```